assets/theme-supplement.css

```diff
@@ -529,4 +539,25 @@
   }
 }
 
+.sc-review {
+  display: grid;
+}
+
+.sc-review_wrapper {
+  gap: var(--sc-spacing-base);
+  padding: 50px;
+  height: -webkit-fill-available;
+  text-align: center;
+  border: var(--sc-border-lightest);
+  box-shadow: 0 1px 4px #8080801c;
```

Comment:
@aaronryematawaran `border`and `box-shadow` can be removed. Please, follow this example:

`<div class="sc-border-lightest sc-box-shadow"></div>`